tests/pthread_cleanup/tests/01-test

```diff
@@ -1,87 +0,0 @@
-#!/usr/bin/env expect
```

Comment:
Why don't port the expect script?

Reply:
The `01-run.py` script does almost the same checks and is consistent with the other test scripts

Reply:
You are right. Wow..  expect is far more verbose than I remember oO